madprops/__init__.py

```diff
@@ -0,0 +1,3 @@
+__doc__ = "DRF library to operate resource's properties as a dictionary"
```

Comment:
This is typical way how we handle cases when we have a single quote inside of string.